Sources/Proxy/Core/View2DProxy/index.js

```diff
@@ -93,6 +127,89 @@ function vtkView2DProxy(publicAPI, model) {
     }
   };
 
+  /**
+   * If cameraResetIsFitting is true, reset camera exactly fit the bounds in the view
+   * Exact fitting requires useParallelRendering, and an active camera
+   */
+  const superLightResetCamera = publicAPI.lightResetCamera;
+  publicAPI.lightResetCamera = (bounds) => {
+    // Always reset camera first to set physicalScale, physicalTranslation and trigger events
+    const initialReset = superLightResetCamera(bounds);
+    if (
+      !model.cameraResetIsFitting ||
+      !model.useParallelRendering ||
+      !initialReset
+    ) {
+      return initialReset;
+    }
+
+    // For each visible prop get the smallest possible convex hull using bounds corners
+    const visiblePoints = [];
+    if (bounds) {
+      // Bounds are given as argument, use their corners
+      vtkBoundingBox.getCorners(bounds, visiblePoints);
+    } else {
+      publicAPI.getRepresentations().forEach((reprenstationProxy) =>
+        reprenstationProxy
+          .getActors()
+          .concat(reprenstationProxy.getVolumes())
+          .forEach((prop) => visiblePoints.push(...getPropCoarseHull(prop)))
+      );
+    }
+
+    // Get the bounds in view coordinates
+    const viewBounds = vtkBoundingBox.reset([]);
+    const viewMatrix = model.camera.getViewMatrix();
+    mat4.transpose(viewMatrix, viewMatrix);
+
+    for (let i = 0; i < visiblePoints.length; ++i) {
+      const point = visiblePoints[i];
+      vec3.transformMat4(point, point, viewMatrix);
+      vtkBoundingBox.addPoint(viewBounds, ...point);
+    }
```

Comment:
didn't you want to use `vtkBoundingBox.transform()` ?

Reply:
Creating a bounding box from points and transforming the bounding box is not equivalent to transforming the points and creating a bounding box from the transformed points